Kitodo/src/main/webapp/WEB-INF/resources/css/kitodo.css

```diff
@@ -506,106 +506,78 @@ Breadcrumb menu
     padding: 0;
 }
 
-.breadcrumb-menu li:first-child a.ui-menuitem-link {
-    background-image: none;
-}
-
-.breadcrumb-menu li:first-child a.ui-menuitem-link span:hover {
-    color: var(--carbon-blue);
-    background-color: var(--pure-white);
-    transition: color .5s, background-color .5s;
+.breadcrumb-menu ul li.ui-breadcrumb-chevron {
+    display: none;
 }
 
-.breadcrumb-menu li:first-child a.ui-menuitem-link span:after {
-    content: '\f015';
-    font-family: 'FontAwesome';
-    text-indent: 0;
-    position: absolute;
-    left: 0;
-    width: 100%;
-    transition: color .5s, background-color .5s;
+.breadcrumb-menu ul li {
+    overflow: visible;
 }
 
-.breadcrumb-menu a.ui-menuitem-link {
+.breadcrumb-menu ul li .ui-menuitem-link {
     background-color: var(--focused-blue);
+    background-image: none;
     border-radius: var(--default-border-radius);
-    width: 36px;
-    height: 36px;
-    line-height: 36px;
+    color: var(--pure-white);
     display: inline-block;
+    font-size: small;
+    height: var(--input-height);
+    line-height: var(--input-height);
+    margin-top: 0;
     text-align: center;
+    text-decoration: none;
+    width: var(--input-height);
 }
 
-.breadcrumb-menu li[role=menuitem] {
-    overflow: visible;
+.breadcrumb-menu ul li .ui-menuitem-text {
+    padding: 0 4px;
+    text-overflow: ellipsis;
 }
 
-.breadcrumb-menu li:not(:first-child) a.ui-menuitem-link {
-    margin-top: 0;
-    color: var(--pure-white);
-    background-color: var(--focused-blue);
-    width: 36px;
-    transition: color .5s, background-color .5s, width .5s;
+.breadcrumb-menu ul li:not(:first-child) {
+    margin-left: 6px;
 }
 
-.breadcrumb-menu li:not(:first-child):hover a.ui-menuitem-link {
-    color: var(--carbon-blue);
-    background-color: var(--pure-white);
-    width: unset;
+.breadcrumb-menu ul li:not(:first-child)::before {
+    background-color: var(--focused-blue);
+    border: solid var(--blue);
+    border-width: 3px 0 0 3px;
+    content: "";
+    height: 12px;
+    left: -8px;
+    position: absolute;
+    transform: translateZ(1px) rotate(-45deg);
+    top: 11px;
```

Comment:
Shouldn't `top` come before `transform` supposing the CSS properties are to be ordered alphabetically?